TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
the nodetypes package is already a dependency of the kickstarter which makes sense.. https://github.com/gerhard-boden/neos-development-collection/blob/master/TYPO3.Neos.Kickstarter/Resources/Private/Generator/Content/Sites.xml#L6 https://github.com/gerhard-boden/neos-development-collection/blob/master/TYPO3.Neos.Kickstarter/composer.json#L9

so no need to complicate this change with that.. if we want to change that, that's a whole different topic,,